### PR TITLE
Improved celery log

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: security/filters.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+from is_core.filters import UIFilterMixin
+
+from .enums import CeleryTaskLogState, CeleryTaskRunLogState
+from .models import CeleryTaskRunLog
+
+
+class CeleryTaskLogStateFilter(UIFilterMixin, SimpleMethodEqualFilter):
+
+    EMPTY_CHOICE = ('', _('--------'))
```
Comment (rubickcz) [inline security/filters.py]: I think that those dashes don't need to be translated.

Comment (matllubos) [reply security/filters.py]: true story

Comment (rubickcz) [reply security/filters.py]: It is not resolved :D I am still seeing it :-)

Comment (matllubos) [reply security/filters.py]: I now

**Pull request merged**